src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/ConfigContext.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater
+{
+    public class ConfigContext
+    {
+        public Dictionary<string, ServiceSpecificContext> ServiceContext { get; }
+
+        public bool NetTcpCertificate { get; }
+
```

Comment:
Try to name bool properties like they are answering a question. For example you already have HasMetadata. I presume this is a bool to specify if NetTcp is using certificates, so maybe NetTcpUsesCertificates. Same with WindowsAuthentication